core/engine/src/builtins/temporal/plain_year_month/mod.rs

```diff
@@ -202,55 +210,172 @@ impl BuiltInConstructor for PlainYearMonth {
     }
 }
 
-// ==== `PlainYearMonth` Accessor Implementations ====
+// ==== `Temporal.PlainYearMonth` static Methods ====
 
 impl PlainYearMonth {
-    fn get_calendar_id(_this: &JsValue, _: &[JsValue], _: &mut Context) -> JsResult<JsValue> {
-        Err(JsNativeError::error()
-            .with_message("not yet implemented.")
-            .into())
+    // 9.2.2 `Temporal.PlainYearMonth.from ( item [ , options ] )`
+    fn from(_: &JsValue, args: &[JsValue], context: &mut Context) -> JsResult<JsValue> {
+        let item = args.get_or_undefined(0);
+        // 1. If Type(item) is Object or Type(item) is String and item is not null, then
+        let inner = if item.is_object() {
```

Comment:
question: Is this up to date?

I currently have the specification showing the below for 9.2.2, which may actually simplify the logic here if so.

`2. If item is an Object and item has an [[InitializedTemporalYearMonth]] internal slot, then`

Reply:
What's the best way to brand check these?

Reply:
Downcasting is not really performance heavy, it's just two u64 comparisons. In this case, since you just need the inner data, you can just do something like:
```rust
if let Some(data) = item.as_object().and_then(JsObject::downcast_ref::<PlainYearMonth>) {
    // Perform ? [GetTemporalOverflowOption](https://tc39.es/proposal-temporal/#sec-temporal-gettemporaloverflowoption)(options).
    // b. Return ! [CreateTemporalYearMonth](https://tc39.es/proposal-temporal/#sec-temporal-createtemporalyearmonth)(item.[[ISOYear]], item.[[ISOMonth]], item.[[Calendar]], item.[[ISODay]]).
}
```
(not typechecked or borrow checked, maybe you'll have to clone if the call to `get_temporal_overflow_option` causes a panic)